src/lib/php/common-sysconfig.php

```diff
@@ -557,6 +557,19 @@ function Populate_sysconfig()
   $desc = _('');
   $valueArray[$variable] = array("'$variable'", "''", "'$prompt'",
     strval(CONFIG_TYPE_PASSWORD), "'SWH'", "5", "'$desc'", "null", "null");
+  $variable = "ScAPIURL";
+  $prompt = _('Scanoss API url');
+  $desc = _('Set URL to SCANOSS API (blank for default osskb.org');
+  $valueArray[$variable] = array("'$variable'",
+    "'https://osskb.org/scan/direct'", "'$prompt'",
+    strval(CONFIG_TYPE_TEXT), "'SSS'", "1", "'$desc'", "'check_url'", "null");
+
+  $variable = "ScToken";
+  $prompt = _('Access token');
+  $desc = _('Set token to access full service (blank for basic scan');
```

Comment:
A cosmetic change required.
```suggestion
  $desc = _('Set token to access full service (blank for basic scan)');
```